tests/test_StellarLocus.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of <REPLACE WHEN RENAMED>.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for the metrics measurement system: wPerp, extinction_corr.
+"""
+
+import unittest
+import numpy as np
+import os
+import astropy.units as u
+
+from lsst.utils import getPackageDir
+from astropy.table import Table
+from metric_pipeline_utils.stellar_locus import stellarLocusResid, calcQuartileClippedStats
+from astropy.coordinates import SkyCoord
+try:
+    from dustmaps.sfd import SFDQuery
+except ModuleNotFoundError:
+    print("The extinction_corr method is not available without first installing the dustmaps module:\n"
+          "$> pip install --user dustmaps\n\n"
+          "Then in a python interpreter:\n"
+          ">>> import dustmaps.sfd\n"
+          ">>> dustmaps.sfd.fetch()\n")
+
+
+DATADIR = os.path.join(getPackageDir('metric_pipeline_tasks'), 'tests', 'data')
+
+
+class StellarLocusTest(unittest.TestCase):
+
+    def load_data(self):
+        '''Helper to load data to process.'''
+        cat_file = 'tract9813_patches55to72_gri_filtered.fits.gz'
+        # catalog = SimpleCatalog.readFits(os.path.join(DATADIR, cat_file))
+        catalog = Table.read(os.path.join(DATADIR, cat_file))
+        return catalog
+
+    def test_extinction_corr(self):
```

Comment:
I think this test should probably test `extinction_corr` instead of SFDQuery.  I think you have all the data you need to do that.  The way this is implemented, we would not catch bugs introduced into our function.

I think it's safe to assume the author of `dustmaps` is doing their own testing, and we should not need to have our own test for their code.